crates/bevy_animation/src/lib.rs

```diff
@@ -325,6 +375,16 @@ pub struct AnimationPlayer {
     transitions: Vec<AnimationTransition>,
 }
 
+// The components that we might need to read or write during animation of each
+// animation target.
```

Comment:
```suggestion
/// The components that we might need to read or write during animation of each
/// animation target.
```

Reply:
This is a private API and won't get rustdoc'd to begin with. Do you still want me to rustdocify it?

Reply:
Yes please: running rustdoc locally will document internal items.